docs/_includes/js/store.js

```diff
@@ -0,0 +1,102 @@
+(function() {
+  var searchBar      = document.querySelector('.SearchBar'),
+      searchBarInput = document.querySelector('.SearchBar input'),
+      searchResults  = document.querySelector('.SearchBar-resultsData'),
+
+      dropdown        = document.querySelector('.Dropdown'),
+      dropdownLabel   = document.querySelector('.Dropdown-selection')
+      dropdownOptions = document.querySelectorAll('.Dropdown .Dropdown-options a'),
+
+      removeIcon = document.querySelector('.remove-icon');
+
+  document.addEventListener('click', function(e) {
+    if (e.target.id != "dropdown-selection" && dropdown.classList.contains('is-shown')) {
+      dropdown.classList.remove('is-shown');
+    }
+  });
```

Comment:
you may want to use `useCapture` flag for this one, if user clicks on an element which stops the event from bubbling/propagating.